projects/payment-lib/src/lib/components/reports/reports.component.ts

```diff
@@ -66,7 +66,7 @@ export class ReportsComponent implements OnInit {
     this.isDateRangeBetnWeek = true;
     this.isDateBetwnMonth = false;
     this.isStartDateLesthanEndDate = false;
-  } else if(reportName && reportName ==='PAYMENT_FAILURE_EVENT' && isDateRangeMoreThanMonth ) {
+  } else if(reportName && (reportName ==='PAYMENT_FAILURE_EVENT'|| reportName ==="TELEPHONY_PAYMENTS") && isDateRangeMoreThanMonth ) {
```

Comment:
Hi Ollie, with isDateRangeMoreThanMonth you are setting isDateBetwnMonth to be true, therefore forcing them to select dates within a month? Is that correct? What if you want two months of reports?

Reply:
This is correct as stated as follows:- When the user selects a date range more than 1 month.
Then under the date selection field a test is displayed 'Please select the date range between 30 days'and the download button report is disabled.

Reply:
Ok, if this is the requirement, that's fine, thanks Ollie.